### PR TITLE
#7-socialmedia-azadeh

### PR DESCRIPTION
**Pull request by azadehroshan**
fake page

## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Review by AnoshMalik — APPROVED** [review]
Great stuff!

**Comment by youweb3** [comment]
all good

File: client/src/pages/Socialmedia.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Socialmedia() {
+  return (
+    <div>Socialmedia</div>
```
Comment (Roman-Hal) [inline client/src/pages/Socialmedia.js:5]: Nice looking line of code. (this is to show how individual comment works.

**Review by Roman-Hal — COMMENTED** [review]
Great job!

**Pull request merged**